src/main/java/networkbook/model/person/Course.java

```diff
@@ -0,0 +1,65 @@
+package networkbook.model.person;
+
+import static java.util.Objects.requireNonNull;
+import static networkbook.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Person's course of study in the network book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidCourse(String)}
+ */
+public class Course {
+
+    public static final String MESSAGE_CONSTRAINTS = "Courses can take any values, and it should not be blank.";
+
+    /*
+     * The first character of the course must not be a whitespace,
+     * otherwise " " (a blank string) becomes a valid input.
+     */
+    public static final String VALIDATION_REGEX = "[^\\s].*";
+
+    public final String value;
```

Comment:
Object-specific fields should not be publicly accessible

Reply:
Meant this to be a class property (hence the static tag) akin to existing implementation. I'll not change this for now for lack of a better alternative on my part, if you have any ideas do implement them!

Reply:
Existing implementation is not the best practice, as there are quite a lot of fields publicly accessible and modifiable. We should only have a function to get the value. You can have a look at the `Identifiable` interface and how `Email` class implements it.

Reply:
Ok! Don't think I'll address this rn, but i'll take not to amend this in a future iteration!